recipe/meta.yaml

```diff
@@ -35,8 +34,9 @@ requirements:
     - packaging
     - pandas >=0.24.2
     - python
-    - scikit-learn >=1.0.0
+    - scikit-learn >=1.2.0
     - scipy
+    - importlib-metadata # [py>=310]
```

Comment:
I don't think `importlib-metadata` is required in run.

Reply:
yes you are right, I have removed that